net-fs/samba/samba-4.16.1.ebuild

```diff
@@ -141,6 +141,7 @@ BDEPEND="${PYTHON_DEPS}
 
 PATCHES=(
 	"${FILESDIR}/${PN}-4.4.0-pam.patch"
+	"${FILESDIR}/${PN}-${PV}-netdb-defines.patch"
```

Comment:
${PN}-${PV} == ${P}, but you should probably just do ${PN}-4.16.1 to match existing style.